### ISSUE TITLE
New terms for EOL TraitBank - sexual maturity

### ISSUE THREAD

**Issue by diatomsRcool** [EcologicalSemantics/ecocore#21]
For new term requests, please provide the following information:

## Preferred term label

Age at sexual maturity

## Textual definition

Age at the first successful reproduction.

There is a related class, onset of fertility, in VT. I'm not sure if we should use that, though. Should we just have sexual maturity as a class? If so, maybe that should be a child of developmental process involved in reproduction GO:0003006

## Suggested parent term

organismal quality PATO:0001995

## Attribution

If you would like a nanoattribution, please indicate your ORCID id
http://orcid.org/0000-0002-2908-3327

**Comment by pbuttigieg** [comment]
>Age at sexual maturity =def. Age at the first successful reproduction.

This doesn't seem right: sexual maturity can precede successful reproduction events. We should make the label more fitting to the definition. If this label/def pair is somehow common usage (:scream:), we can include it as a related synonym of a more controlled class.

Also, "age at sexual maturity" doesn't really sound like a PATO:quality, but more like some sort of IAO data item. A quality mini-hierarchy would be something like:
* sexual maturity quality
  * sexually mature
  * sexually immature

(regardless of the age thing, we should have the mini-hierarchy above somewhere in PATO or an ECOCORE extension of PATO for better granularity)

Not an immediate fan, but I think we could combine this with PATO:[`chronological age`](http://purl.obolibrary.org/obo/PATO_0001486) as a superclass to come up with something like "chronological age at sexual maturity".

>There is a related class, onset of fertility, in VT. I'm not sure if we should use that, though. Should we just have sexual maturity as a class? If so, maybe that should be a child of developmental process involved in reproduction GO:0003006

They sound very similar and an exact synonym AP may be appropriate. If the first label/def issue is cleared up, it will help the decision. The [superclass you suggest](http://purl.obolibrary.org/obo/GO_0003006) sounds right.

**Comment by diatomsRcool** [comment]
I think there are two things happening here. There is sexual maturity and then there is age at sexual maturity. I'm checking back with EOL to double check their definition of "age at sexual maturity". I like the "chronological age at sexual maturity" idea.

**Comment by jhammock** [comment]
yes, several concepts above, I think. We do use a fistful of terms in this area. 

For lifestages we use PATO terms, one from EFO (http://www.ebi.ac.uk/efo/EFO_0001272) and some ad hoc eol placeholder terms. We don't have sexual maturity in a separate branch; just all lifestage categories together; there are a couple of short hierarchies in those.

For the chronological concept we've got data for both age at maturity and age at first birth. Mostly ad hoc terms for both, though there's one from PolyTraits (http://polytraits.lifewatchgreece.eu/terms/MAT), which might stick around.

**Comment by jhammock** [comment]
@cmungall The chronological terms? That might make sense. They don't construct in the PATO+UBERON way that I'm familiar with. http://purl.obolibrary.org/obo/PATO_0001701 + http://purl.obolibrary.org/obo/PATO_0000011 somehow, maybe?

**Comment by cmungall** [comment]
I'd use the uberon stage classes here for things like sexually mature stage

**Comment by jhammock** [comment]
that makes sense- but I'm not sure which UBERON term is appropriate. Maybe http://purl.obolibrary.org/obo/UBERON_0018241

**Comment by diatomsRcool** [comment]
I'll sort this out once I get in to where I can see how EOL is using these terms. Adding a chronological age term to a life stage term in OBA is likely the correct way to do things.

**Comment by diatomsRcool** [comment]
I think EOL needs to clarify some of their definitions a bit more and I will work with them on that. We need two separate terms for EOL
Chronological age at sexual maturity: Make this in OBA by combining PATO chronological age with UBERON post-juvenile adult stage

AND

Chronological age at first reproduction: UBERON does not have a life stage that represents this, but EOL appears to have these data.

@jhammock can we chat via gitter?

**Comment by diatomsRcool** [comment]
https://github.com/obophenotype/bio-attribute-ontology/issues/30
